canonical-web-and-design/charmhub.io
# Change layout to 82rem

## Done

- Change layout to 82rem
- Create a new layout for the store page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://localhost:8045/store and see it looks and works fine for all screen sizes
- Visit other pages and see everything still looks good: [home](http://localhost:8045) [about](http://localhost:8045/about) [topics](http://localhost:8045/topics/kubernets)


## Issue / Card

Fixes #279
